seed/Page.js

```diff
@@ -12,3 +12,14 @@ module.exports.createMainImageRecords = (id) => {
     })
     .join('');
 };
+
+module.exports.createCouchRecord = (id) => {
+  const randomPage = randomPageSelector();
+  const mainImages = [...new Array(8)].map((currentelement, index) => {
+    return JSON.stringify({
+      main: `https://steam-fec.s3.amazonaws.com/steam${randomPage}/main-${randomPage}-${index}.jpg`,
+      thumb: `https://steam-fec.s3.amazonaws.com/steam${randomPage}/thumb-${randomPage}-${index}.jpg`
+    })
+  })
+  return `${id}\thttps://steam-fec.s3.amazonaws.com/steam${randomPage}/header-${randomPage}.jpg\t[${mainImages}]\n`
```

Comment:
I want to know what \t does, I know that was a big discovery for you from the slack chat

Reply:
`\t` is a tab. i was trying to insert a string with commas into mysql but i was running into a problem. the default column delimeter/separator for `LOAD DATA LOCAL` is a comma `,` so it thought every comma was a new column/value. 

`'value','some,string,with,commas'` => 5 columns

i was able to set the delimeter/separator to `\t` and separate my columns with tabs.

`'value'    'some,string,with,commas'` => 2 columns